src/codewhisperer/service/securityIssueHoverProvider.ts

```diff
@@ -41,6 +41,19 @@ export class SecurityIssueHoverProvider implements vscode.HoverProvider {
         return new vscode.Hover(contents)
     }
 
+    public removeFixedIssue(filePath: string, fixedIssue: CodeScanIssue) {
```

Comment:
We can probably remove this once my changes are in, since applying the fix should trigger the document change event and clean up the issues.

Reply:
yes, we no longer need this method. removing it.